cpp/modules/luma.gl/garrow/src/array.cc

```diff
@@ -34,8 +34,15 @@ Array::~Array() {
 }
 
 void Array::setData(const std::shared_ptr<arrow::Array>& data) {
+  auto vertexFormatOptional = vertexFormatFromArrowType(data->type());
+  if (!vertexFormatOptional.has_value()) {
+    throw std::runtime_error("Unsupported data format");
+  }
+  auto vertexFormat = vertexFormatOptional.value();
+  auto vertexSize = getVertexFormatSize(vertexFormat);
```

Comment:
See my comment on conversion - I am not sure we need to expose a wgpu type here.